### DISCUSSION TITLE
Plugins - needed packages

### DISCUSSION THREAD

**Discussion by brainbow1** [zim-desktop-wiki/zim-desktop-wiki#1666] [Q&A]
Hello!

I tried to find the right packages that I need under Debian, for using the Sequence Diagram Plugin... 
The way with "sudo easy_install seqdiag" doesnt work, also "sudo pip install seqdiag" doesnt work..

when I do the last mentioned:
`debian@debian:~$ sudo pip install seqdiag
Requirement already satisfied: seqdiag in /usr/lib/python3/dist-packages (2.0.0)
`
But in zim it shows that the dependency is not satisfied...

Also I want to activate the LaTex / insert equation plugin, which says that I need latex and dvipng installed...

dvipng was no problem to install, but when I search for latex in debian packages there are lots of different pakcages... which should I choose?
https://packages.debian.org/search?suite=stable&section=all&arch=any&searchon=names&keywords=latex

Best regards!

**Comment by introt** [comment]
Hi!

Your `seqdiag` executable is likely called `seqdiag3` and thus isn't detected automatically -- we're tracking that at  #1613 -- so you need to symlink it to `seqdiag` for it to be picked up. You can see [here](https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/1609#discussioncomment-1373690) how that's done - if the path is different on your system, use `which seqdiag3` to find the right one. Link destination doesn't matter, as long as it's in `$PATH`.

If you're not concerned about having a bunch of packages around and don't need Chinese characters, from that list `latex-make` should pull enough packages to render some tex.

Cheers!
